src/main/resources/localstack/s3/sites/sites.json

```diff
@@ -2,27 +2,32 @@
   {
     "id": 999,
     "name": "Special",
-    "enabled": true
+    "enabled": true,
+    "clientTypes": ["DSP", "PUBLISHER", "DATA_PROVIDER", "ADVERTISER"]
   },
   {
     "id": 123,
     "name": "DSP",
-    "enabled": true
+    "enabled": true,
+    "clientTypes": ["DSP"]
   },
   {
     "id": 124,
     "name": "Publisher",
-    "enabled": true
+    "enabled": true,
+    "clientTypes": ["PUBLISHER"]
   },
   {
     "id": 125,
     "name": "Data Provider",
-    "enabled": true
+    "enabled": true,
+    "clientTypes": ["DATA_PROVIDER"]
   },
   {
     "id": 126,
     "name": "Advertiser",
-    "enabled": true
+    "enabled": true,
+    "clientTypes": ["ADVERTISER"]
```

Comment:
I think we need some sites that have no clientTypes assigned yet, right?

Reply:
Yea, good point! sites added